theano/gpuarray/c_code/dnn_sptf_grid.c

```diff
@@ -0,0 +1,129 @@
+#section support_code_struct
+
+cudnnSpatialTransformerDescriptor_t APPLY_SPECIFIC(sptf);
+
+#section init_code_struct
+
+cudnnStatus_t APPLY_SPECIFIC(err) = CUDNN_STATUS_SUCCESS;
```

Comment:
You can put code into braces as you do in `dnn_sptf_gi.c`, so that you won't need to use `APPLY_SPECIFIC(err)`.